.github/workflows/bf-review.yml

```diff
@@ -55,6 +55,9 @@ jobs:
             context: ./frontend
             dockerfile: ./frontend/benefit/applicant/Dockerfile
             port: 3000
+    concurrency:
+      group: ${{ env.K8S_NAMESPACE }}-${{ matrix.service }}
+      cancel-in-progress: false
```

Comment:
I think there's no need to block build concurrency, as each build created different image anyway.